docs/user_guide.md

```diff
@@ -5,16 +5,20 @@
 4. Profit!
 
 ## Dependencies
-1. Git
-2. Gradle
+1. **JDK `1.8.0_60`**  or later <br>
+> This app only works with Java 8 version. <br>
```

Comment:
Let's not use 'quote' formatting for things that are not really quotes. 
No need for `<br>` if you use correct indentation. e.g.,

1. **JDK `1.8.0_60`**  or later
   This app only works with Java 8 version.

```
1. **JDK `1.8.0_60`**  or later
   This app only works with Java 8 version.
```

Reply:
Hmm, I think it only works in comment. It works even without indentation.

Reply:
Try this (extra line break)
```
1. **JDK `1.8.0_60`**  or later

   * This app only works with Java 8 version.
```

BTW, the app doesn't work for Java 9, 10?

Reply:
I mean the code quote version works but not this one without `<br>` 

1. **JDK `1.8.0_60`**  or later
   This app only works with Java 8 version.

> BTW, the app doesn't work for Java 9, 10?

Yup, seems like the code is build in Java 8 so we will need to upgrade to Java 9 / 10 for it to work with latest java jdk. Meanwhile the workaround is by overwriting the system variable using
`set JAVA_HOME=JAVA_8_JDK_DIRECTORY` in windows.

Reply:
>A java 8 app should work within Java 9 or 10 unless we use some backward incompatible library or API. Did you try with later versions?

Yup, my default jdk is java 9. I'm unable to build it with java 9 but I have not investigate on the cause, so currently I'm still using the workaround.

Reply:
![image](https://user-images.githubusercontent.com/19277206/39872619-25ee20a6-549b-11e8-9048-c8ea3991c9ce.png)

Seems like its due to gradle out-of-date

Reply:
Got it. In this PR we can require users to use Java 8. Hopefully, we can make it compatible with Java 9 soon.